src/syft/lib/__init__.py

```diff
@@ -142,20 +145,26 @@ def _load_lib(*, lib: str, options: TypeDict[str, TypeAny] = {}) -> None:
             _regenerate_unions(lib_ast=lib_ast, client=client)
 
 
-def load(lib: str, options: TypeDict[str, TypeAny] = {}) -> None:
+def load(
```

Comment:
Could you add in the type hints that we want specifically strings?
`List[str], Set[str]` etc.

It has to be clear for the user that it shouldn't pass the module itself.

Reply:
Done.